homeassistant/components/conversation/default_agent.py

```diff
@@ -188,29 +186,25 @@ def _filter_state_changes(self, event_data: core.EventStateChangedData) -> bool:
     @core.callback
     def _listen_clear_slot_list(self) -> None:
         """Listen for changes that can invalidate slot list."""
-        assert self._unsub_clear_slot_list is None
-
-        self._unsub_clear_slot_list = [
-            self.hass.bus.async_listen(
-                ar.EVENT_AREA_REGISTRY_UPDATED,
-                self._async_clear_slot_list,
-            ),
-            self.hass.bus.async_listen(
-                fr.EVENT_FLOOR_REGISTRY_UPDATED,
-                self._async_clear_slot_list,
-            ),
-            self.hass.bus.async_listen(
-                er.EVENT_ENTITY_REGISTRY_UPDATED,
-                self._async_clear_slot_list,
-                event_filter=self._filter_entity_registry_changes,
-            ),
-            self.hass.bus.async_listen(
-                EVENT_STATE_CHANGED,
-                self._async_clear_slot_list,
-                event_filter=self._filter_state_changes,
-            ),
-            async_listen_entity_updates(self.hass, DOMAIN, self._async_clear_slot_list),
-        ]
+        self.hass.bus.async_listen(
+            ar.EVENT_AREA_REGISTRY_UPDATED,
+            self._async_clear_slot_list,
+        )
+        self.hass.bus.async_listen(
+            fr.EVENT_FLOOR_REGISTRY_UPDATED,
+            self._async_clear_slot_list,
+        )
+        self.hass.bus.async_listen(
+            er.EVENT_ENTITY_REGISTRY_UPDATED,
+            self._async_clear_slot_list,
+            event_filter=self._filter_entity_registry_changes,
+        )
+        self.hass.bus.async_listen(
+            EVENT_STATE_CHANGED,
+            self._async_clear_slot_list,
+            event_filter=self._filter_state_changes,
+        )
```

Comment:
Are these persistent when the integration is reloaded? If not we probably should cleanup the handler when unloading.

Reply:
I've reverted this change.